proc_echo_mw.py

```diff
@@ -1,143 +1,101 @@
 from pyspecdata import *
-from utility import dBm2power
 from scipy.optimize import leastsq,minimize,basinhopping,nnls
-from proc_scripts import hermitian_function_test, zeroth_order_ph
+from proc_scripts import *
+from proc_scripts import postproc_dict
 from sympy import symbols
+init_logging(level='debug')
 rcParams["savefig.transparent"] = True
-def expand_limits(thisrange,d):
-    thisrange = list(thisrange)
-    full_range = d.getaxis('t2')[r_[0,-1]]
-    retval = array([thisrange[j] if thisrange[j] is not
-            None else full_range[j] for j in [0,-1]])
-    print(repr(retval))
-    m = mean(retval)
-    d = m-retval
-    retval = 3*d+m
-    sgn = [-1,1] # greater than or less than
-    return tuple(full_range[j] if
-            retval[j]*sgn[j] > full_range[j]*sgn[j]
-            else
-            retval[j] for j in range(2))
-def draw_limits(thisrange,d):
-    full_range = d.getaxis('t2')[r_[0,-1]]
-    dw = diff(d.getaxis('t2')[:2]).item()
-    print("I find the full range to be",full_range)
-    sgn = [-1,1] # add or subtract
-    pairs = [[thisrange[j],full_range[j]+0.5*dw*sgn[j]] for j in range(2)]
-    pairs[0] = pairs[0][::-1] # flip first
-    my_xlim = gca().get_xlim() # following messes w/ xlim for some reason
-    for j in pairs:
-        if None not in j:
-            print("drawing a vspan at",j)
-            axvspan(j[0],j[1],color='w',alpha=0.5,linewidth=0)
-    gca().set_xlim(my_xlim)
-class fl_mod (figlist_var):
-    def side_by_side(self,plotname,s,thisrange):
-        """a bit of a hack to get the two subplots into
-        the figure list -- also a good test for objective
-        figure list -- for each slice out 3x thisrange, and then
-        show the lines for thisrange"""
-        thisfig,(ax1,ax2) = subplots(1,2)
-        fl.next(plotname, fig=thisfig)
-        sca(ax1)
-        forplot = s['t2':expand_limits(thisrange,s)]
-        fl.image(forplot)
-        print("drawing limits",thisrange)
-        draw_limits(thisrange,forplot)
-        sca(ax2)
-        fl.image(forplot.C.cropped_log())
-        draw_limits(thisrange,forplot)
-        title('cropped log')
-        return
+
 fl = fl_mod()
 t2 = symbols('t2')
-# notice how I dramatically simplified the following
-# going forward, this is how we should deal with filenames
-# (that was an earlier commit... then I use the list of
-# tuples to specify the ranges for frequency and time)
 
-# to use: as a rule of thumb, make the white boxes
-# about 2x as far as it looks like they should be
```

Comment:
why is this commentary deleted?

Reply:
I forget what this pertained to. I may have just seen the comment and thought it was an old command.

Reply:
no, this explains how to use the script, actually

Reply:
I think I put this back in the right place?